apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss

```diff
@@ -40,6 +52,14 @@
             s {
                 color: var(--color-gray);
             }
+
+            @media screen and (max-width: 1299px) {
+                margin-bottom: 6px !important;
+
+                .plan-price {
+                    font-size: 18px !important;
```

Comment:
why use static font ?

We have available font sizes

--font-size-16 and --font-size-20

Reply:
done